core/tsdx.config.js

```diff
@@ -17,7 +18,8 @@ module.exports = {
         // A workaround for babel-plugin-transform-async-to-promises used by tsdx
         // injecting helper code with `const` and producing invalid ES5 code.
         config.plugins.push(replace({
-            'const ': 'var '
+            'const ': 'var ',
+            'LOCAL_VERSION': version,
```

Comment:
This seems to be a thing inside of a hack. Ideally, this hack shouldn't even be there.
It would probably be less hacky to do version the same way __DEV__ works. It'll be some globally declared variable available anywhere.

Reply:
but having __DEV__ relies on some bundler injecting that during the build process to all usages, the only difference here is that there is no need for a global type declaration.

But I can go the way of using a global variable too and replace at build